src/components/Card.tsx

```diff
@@ -11,15 +11,14 @@ function Card(): JSX.Element {
 }
 
 function CharacterCard({ costume }: { costume: CostumeInfo }): JSX.Element {
+
   return (
-    <Link href={"/characters/" + costume.character + "/" + costume.name.en.replace(" ", "-")}>
+    <Link href={`/characters/${costume.character}/${costume.name.en.replace(" ", "-")}`}>
```

Comment:
We should use slugify here

Reply:
Ok what would that look like?
This?
`{`${slugify(`/characters/${costume.character}/${costume.name.en}}`)}`}`
Or this?
`{slugify(`/characters/${costume.character}/${costume.name.en}}`)}`

Reply:
Markdown messed with formatting

![image](https://user-images.githubusercontent.com/5669203/131254143-0d76ec17-c16e-4a74-8bbb-df6f06604948.png)

Reply:
```jsx
<Link href={`/characters/${costume.character}/${slugify(costume.name.en, { lower: true, remove: /[*+~.()'"!:@]/g })}`}>
```

Reply:
Now that I'm looking at it. I want to slugify characters as well. Should this be 2 separate slugifys? or one big one

Reply:
Why characters ?

Reply:
Well some might also have spaces, I can only think of one right now "The World Ender" but maybe more in the future?

Reply:
ha you meant `costume.character` yeah

Reply:
yeah, so two slugifys? or one big one. Not really sure what's better practice

Reply:
Well yeah two call for slugify, I don't think slugify accept two slugify at once with the same parameters

Reply:
well one would look like this..
```${slugify(`${costume.character}/${costume.name.en}`, { lower: true, remove: /[*+~.()'"!:@]/g })}`}>```

Reply:
Oh yeah, might work? Idk lol

Reply:
Ok it works, but also doesn't. Apparently the character urls are case sensitive. Going to update without `lower` for now, but maybe that's something we can do to character urls later on. (Make them case-insensitive, didn't know they would be in the first place :/)

Reply:
I'll do it no worry